third_party/blink/renderer/modules/cobalt/h5vcc_metrics/h_5_vcc_metrics.cc

```diff
@@ -108,10 +109,6 @@ void H5vccMetrics::OnDisable(ScriptPromiseResolver* resolver) {
   resolver->Resolve();
 }
 
-void H5vccMetrics::OnIsEnabled(ScriptPromiseResolver* resolver, bool result) {
```

Comment:
What replaced thsi?

Reply:
I think the idea was to have this be the the bottom-half of 
`isEnabled()` but currently said method (l.62) just returns 
the value of `is_reporting_enabled_` so this is effectively
dead code.